### PR TITLE
Fix wrong demangling of tiff swab16 bit data

### PR DESCRIPTION
**Pull request by ghost**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [x] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the documentation and the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

...

**Test plan**

```
rz-test test/db/formats/coff
```

**Closing issues**

see  #851

### CODE REVIEW

File: librz/bin/mangling/swift-sd.c [hunk 1]
```diff
@@ -263,8 +259,8 @@ RZ_API char *rz_bin_demangle_swift(const char *s, bool syscmd) {
 			}
 			break;
 		case 'I': // interfaces
-			/* TODO */
-			break;
+			// TODO
+			return NULL;
```
Comment (XVilka) [inline librz/bin/mangling/swift-sd.c:263]: Is this the exact change that fixes test?

**Comment by XVilka** [comment]
cc @pelijah

**Comment by XVilka** [comment]
Probably worth to squeeze into 0.2.0 too.

File: librz/bin/mangling/swift-sd.c (hunk 1 above)
Comment (ghost) [reply librz/bin/mangling/swift-sd.c:263]: Yes

**Review by XVilka — APPROVED** [review]

**Comment by XVilka** [comment]
This whole code should be rewritten in the future.

**Review by wargio — APPROVED** [review]

**Comment by wargio** [comment]
> This whole code should be rewritten in the future.

open an issue.

**Comment by ghost** [comment]
I can merge (rebase)?

**Pull request merged**